geo/src/test/java/org/odk/collect/geo/support/Fixtures.kt

```diff
@@ -0,0 +1,20 @@
+package org.odk.collect.geo.support
+
+import android.R
+import org.odk.collect.geo.MappableSelectItem
+
+object Fixtures {
+    fun mappableSelectItem(): MappableSelectItem {
```

Comment:
We use builders a lot in Java, but I like this pattern a lot more in Kotlin - provide a basic "fixture" object and allow tests to customize through `copy()`.